main.go

```diff
@@ -569,10 +569,14 @@ func initMajordomo(ctx context.Context) (majordomo.Service, error) {
 		}
 	}
 
-	if viper.GetString("majordomo.gsm.credentials") != "" {
+	if viper.GetString("majordomo.gsm.project") != "" {
+		var gsmCredentialPath string
```

Comment:
Can we use `gsmCredentialsPath` here as we use the plural everywhere else?  Thanks.

Reply:
fixed